source/lib/rocprof-sys/library/rocprofiler-sdk.cpp

```diff
@@ -222,22 +222,24 @@ save_args(rocprofiler_callback_tracing_kind_t /*kind*/, int32_t /*operation*/,
 auto&
 get_marker_pushed_ranges()
 {
-    static thread_local auto _v = std::vector<tim::hash_value_t>{};
+    static thread_local auto _v =
+        std::vector<std::pair<tim::hash_value_t, rocprofiler_timestamp_t>>{};
     return _v;
 }
 
 auto&
 get_marker_started_ranges()
 {
-    static thread_local auto _v = std::vector<tim::hash_value_t>{};
+    static thread_local auto _v =
+        std::vector<std::pair<tim::hash_value_t, rocprofiler_timestamp_t>>{};
     return _v;
 }
 
 template <typename CategoryT>
 void
 tool_tracing_callback_start(CategoryT, rocprofiler_callback_tracing_record_t record,
                             rocprofiler_user_data_t* /*user_data*/,
-                            rocprofiler_timestamp_t /*ts*/)
+                            rocprofiler_timestamp_t ts)
 {
```

Comment:
```suggestion
    // Required because of how some compilers handle templates. This may result in an
    // "unused variable" warning.
    (void) ts;

```
Because of how older compilers use templates, you'll want to make sure `ts` appears "used".